server/config.ts

```diff
@@ -45,7 +45,7 @@ export default {
   branchName: get('GIT_BRANCH', 'xxxxxxxxxxxxxxxxxxx', requiredInProduction),
   production,
   https: production,
-  staticResourceCacheDuration: 20,
```

Comment:
20 seconds is really low, this was changed in the Typescript Template [here](https://github.com/ministryofjustice/hmpps-template-typescript/pull/178/files), I must have missed it in merge conflicts when pulling template updates in